com.microsoft.mrtk.input/Visualizers/ControllerVisualizer/ControllerModelLoader.cs

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using UnityEngine;
+using System.Threading.Tasks;
+
+#if MROPENXR_PRESENT && (UNITY_STANDALONE_WIN || UNITY_WSA || UNITY_ANDROID) && GLTFAST_PRESENT && KTX_PRESENT
+using Microsoft.MixedReality.OpenXR;
+using GLTFast;
+#endif // MROPENXR_PRESENT && (UNITY_STANDALONE_WIN || UNITY_WSA || UNITY_ANDROID) && GLTFAST_PRESENT && KTX_PRESENT
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// A helper class which loads and caches controller models fetched from the controller's platform sdk
+    /// </summary>
+    public class ControllerModelLoader
+    {
+        /// <summary>
+        /// A dictionary which caches the controller model gameobject associated with a specified model key
+        /// </summary>
+        public static Dictionary<ulong, GameObject> ControllerModelDictionary = new Dictionary<ulong, GameObject>();
```

Comment:
Does this need to be `public`?

Also nit on the capitalization of this field (unless it needs to be `public` and could become a property!)

Reply:
Making it private and with pascalcase